torch/testing/_internal/common_distributed.py

```diff
@@ -39,6 +39,8 @@ def wrapper(*args, **kwargs):
         if not torch.cuda.is_available():
             sys.exit(TEST_SKIPS["no_cuda"].exit_code)
         if torch.cuda.device_count() < int(os.environ["WORLD_SIZE"]):
+            message = "Need at least {} CUDA devices".format(2)
```

Comment:
should this be?

```
message = "Need at least {} CUDA devices".format(os.environ["WORLD_SIZE"])
```

Reply:
hi @mrshenli ，thanks for point it out， I'll modify it.